src/tests/ftest/container/autotest.py

```diff
@@ -15,23 +15,19 @@ class ContainerAutotestTest(TestWithServers):
     :avocado: recursive
     """
 
-    def __init__(self, *args, **kwargs):
-        """Initialize object."""
-        super().__init__(*args, **kwargs)
-        self.daos_cmd = None
-
     def test_container_autotest(self):
         """Test container autotest.
 
-        :avocado: tags=all,full_regression,daily,hw,small,quick,autotest
+        :avocado: tags=all,full_regression,daily_regression
```

Comment:
Is there any reason we have both full_regression and daily_regression? I think if you have daily, you wouldn't need full.

Reply:
I'm not 100% sure, I'll investigate it and I don't think this really blocks the merge.